tst/testinstall/opers/CyclesFromList.tst

```diff
@@ -0,0 +1,33 @@
+gap> START_TEST("CyclesFromList")
```

Comment:
This is missing a semicolon, hence the test fails.

You can test the .tst file locally using `Test()`, just do
```
Test("tst/testinstall/opers/CyclesFromList.tst");
```
(adjust the path depending on from where you run GAP)